multiregion: replace TODO and unimplemented with issue numbers

otan opened the pull request:
In the spirit of Milestone planning, replace all remaining multiregion
TODOs with issue numbers.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/59633)
<!-- Reviewable:end -->

otan committed:
multiregion: replace TODO and unimplemented with issue numbers

In the spirit of Milestone planning, replace all remaining multiregion
TODOs with issue numbers.

Release note: None

ajstorm reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

One issue (https://github.com/cockroachdb/cockroach/issues/59630) wasn't on the board, so I added it there.  I'm thinking this one is MVP, so please let me know if you disagree.

> Reviewed 6 of 7 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/59633#-:-MSV90fI9lYqvq1kOLG_:b2xhjhx)* status: :shipit: complete! 1 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

otan commented:
thx

bors r=ajstorm
